model/sources/IDU/README.md

```diff
@@ -0,0 +1,47 @@
+# IDU
+
+API: https://helix-tools-api.idmcdb.org/external-api/
+Current Usage: https://www.internal-displacement.org/internal-displacement-updates/
+
+## Collection: `idu-events`
+
+Collection: idu-events
+A STAC collection hold all the IDU events.
+
+Name: Internal displacement updates
+
+Code: IDU
+
+Source organization: Internal Displacement Monitoring Center (IDMC)
+
+Source organization type: Regional Intergovernmental Organization
+
+Source organization email: info@idmc.ch
+
+Source code: IDMC
+
+Source URL: https://helix-tools-api.idmcdb.org/external-api/
+
+Source for: event, impact
+
+previous implementation (R): none
+
+
+| STAC field                         | IDU Field                            | Remarks                        |
+| ---------------------------------- | ------------------------------------ | ------------------------------ |
+| id                                 | id                                   | Figure id                      |
+| collection                         | `idu-events`                         | Collection name for STAC       |
+| title                              | event_name                           | Name of the event              |
+| description                        | standard_popup_text                  |                                |
+| datetime                           | displacement_date                    | Start date of the event        |
+| geometry                           | type: "point", coordinates: centroid | centroid of the event location |
+| monty:country_codes[0]             | iso3                                 |                                |
+| monty:event_codes                  | event_codes                          | Might be empty                 |
+| properties.locations               | locations_name                       |                                |
+| properties.event_start_date        | event_start_date                     |                                |
+| properties.event_end_date          | event_end_date                       |                                |
+| properties:displacement_start_date | displacement_start_date              |                                |
+| properties:displacement_end_date   | displacement_end_date                |                                | 
+| properties.sources                 | sources                              |                                |
+| properties.event_id                | event_id                             |                                |
+| properties.hazard_subtype          | hazard_subtype                       |                                |
```

Comment:
Those properties do not exist in STAC. Only STAC properties should be listed here. Some remarks:
- what are event_codes?
- locations should go in keywords
- event_start_date and event end_date must be mapped to start_datetime and end_datetime for events items
- displacement_start_date and displacement_end_date must be mapped to start_datetime and end_datetime for impact items
- sources may be added in the links
- what is event id and how is it different from id?
- hazard_subtype should be mapped to harzard_codes

Reply:
Where do we include other additonal properties? @emmanuelmathot

Reply:
The mapping is indeed the hardest work. The [core](https://github.com/radiantearth/stac-spec) fields and [extensions](https://github.com/stac-extensions) help to find the right one. For location, all "free text" element such as location should be put in the general [keywords](https://github.com/radiantearth/stac-spec/blob/master/commons/common-metadata.md#basics) array field as it is fully supported by pgstac for token search. state and end dates always goes to start_datetime and end_datetime [fields](https://github.com/radiantearth/stac-spec/blob/master/commons/common-metadata.md#date-and-time-range).